common/app/views/support/cleaner/AmpEmbedCleaner.scala

```diff
@@ -60,6 +60,24 @@ case class AmpEmbedCleaner(article: Article) extends HtmlCleaner {
 
   }
 
+  def cleanAmpSoundcloud(document: Document) = {
+    document.getElementsByClass("element-audio").filter { audioElement: Element =>
+      audioElement.getElementsByTag("iframe").length != 0
```

Comment:
actually, I'm less of a fan of negating: `!.....................................................isEmpty` because we're splitting the "not" and the "empty"
some people even go for `!!!.................................................isEmpty` just to make the `!!!` more obvious